packages/electrumAnalyzer/electrumAnalyzer.0.3.4/opam

```diff
@@ -0,0 +1,24 @@
+opam-version: "1.2"
+maintainer: "David Chemouil <david.chemouil+electrum@onera.fr>"
+authors: ["David Chemouil" "Julien Brunel" "Denis Kuperberg"]
+homepage: "https://forge.onera.fr/projects/electrum"
+bug-reports: "https://forge.onera.fr/projects/electrum"
+license: "GPL-3+"
+dev-repo: "https://git.onera.fr/electrum.git"
+build: [
+  ["./configure" "--prefix=%{prefix}%"]
+  [make]
+]
+install: [make "install"]
+remove: [
+  ["rm" "-f" "%{bin}%/electrumAnalyzer"] {os = "darwin" | os = "linux"}
+]
+depends: [
+  "batteries" {build}
+  ("cmdliner" {build & = "0.9.7"} | "cmdliner" {build & = "0.9.6"} |
+   "cmdliner" {build & = "0.9.5"})
+  "menhir" {build & >= "20150720"}
+  "ocamlfind" {build & >= "1.5.5"}
+  "pprint" {build}
```

Comment:
On Tue, 3 Nov 2015 00:25:57 -0800, David Chemouil wrote:
> 
> Running oasis2opam does constrain ocamlfind to be >- 1.5.5, BTW, not 1.5.

This is because you required so in your _oasis file.

You may want to use `oasis2opam --local` in your sources to see the opam dir created (be aware that, if you use dynamic mode for development, it will add a dependency on `oasis`).